testkit/Dockerfile

```diff
@@ -10,6 +10,8 @@ RUN apt-get update && \
         npm \
     && rm -rf /var/lib/apt/lists/*
 
+RUN npm install -g n
+RUN n 10
```

Comment:
is this a typo?

Reply:
Not this time. 

https://www.npmjs.com/package/n

Reply:
what a great package name #not 😅
interesting, I've only used `nvm` for this.
TIL!

Reply:
yeah, It remembers the var names I used to use in algorithms classes.